WooCommerce/Classes/ViewRelated/Upgrades/UpgradesView.swift

```diff
@@ -457,6 +479,12 @@ struct UpgradeWaitingView: View {
 
             Spacer()
         }
+        .onAppear {
+            onViewLoaded()
+        }
+        .onDisappear {
+            onViewDismissed()
+        }
```

Comment:
This looks like the cause of the over-logging of the events... `appear` and `disappear` will happen much more often than we want to track.